### DISCUSSION TITLE
Update the 'connecting to testnet' example

### DISCUSSION THREAD

**Discussion by Dhaiwat10** [FuelLabs/fuels-ts#1690] [Ideas]
As mentioned in [this PR's conversation](https://github.com/FuelLabs/fuels-ts/pull/1677#issue-2096797295), the example on the page itself demonstrates initializing a wallet on the testnet and does not actually interact with any contracts on the testnet or with the testnet node itself. 

I believe this example should show users how they can connect to and interact with a live beta-5 contract.

**Comment by arboleya** [comment]
Could we reuse another example we already have, only changing the node URL to point to live/beta5?

**Reply by Dhaiwat10** [reply]
I'm not aware of any. If there is one, we can re-use. I'll investigate this

**Reply by petertonysmith94** [reply]
@Dhaiwat10 did you investigate this one?

**Comment by petertonysmith94** [comment]
@Dhaiwat10 can this be closed
